core/src/test/scala/org/apache/spark/SparkConfSuite.scala

```diff
@@ -242,7 +242,8 @@ class SparkConfSuite extends SparkFunSuite with LocalSparkContext with ResetSyst
   }
 
   test("deprecated configs") {
-    val conf = new SparkConf()
+    // Set `loadDefaults` to be false, so avoid system props's noises.
```

Comment:
I'd remove this comment, I'm not sure what the 'noise' is here. It's really a workaround for your env.

Reply:
Thanks @srowen 's replies. I resolved the comments.